bin/sl-pmctl.js

```diff
@@ -432,9 +432,16 @@ function extra() {
 
 function remoteRequest(pmctl, cmd, callback) {
   var endpoint = url.parse(pmctl);
-
   if (!endpoint.protocol) {
-    return client.request(pmctl, cmd, callback);
+    pmctl = path.resolve(pmctl);
+
+    if (fs.existsSync(pmctl)) {
+      pmctl = 'http+unix://' + pmctl;
+      remoteHttpRequest(pmctl, cmd, callback);
```

Comment:
This could probably be simplified. I wrote `maybeTunnel()` so that is is a no-op for non-ssh URLs so that you don't need to wrap it in a conditional.

```js
var endpoint = url.parse(pmctl);
if (!endpoint.protocol) {
  if (fs.existsSync(pmctl)) {
    pmctl = 'http+unix://' + pmctl;
  } else {
    console.error('Control socket %s not found', pmctl);
  }
}
// treat http, http+unix, http+ssh, etc. all the same
maybeTunnel(pmctl, sshOpts, ....
```